sympy/core/assumptions_generated.py

```diff
@@ -37,42 +37,33 @@
 ] # defined_facts
 
 
-full_implications = dict( [
-    # Implications of algebraic = True:
-    (('algebraic', True), set( (
+full_implications = { ('algebraic', True): {
```

Comment:
This file is also automatically generated. It should only be changed by the script that generates it.